stdlib/public/Darwin/Accelerate/vDSP_Reduction.swift

```diff
@@ -252,6 +252,56 @@ extension vDSP {
             return output
     }
     
+    /// Returns sum of elements and sum of elements' squares; single-precision.
+    ///
+    /// - Parameter vector: The input vector.
+    /// - Returns: The sum of values and the sum of squares in `vector`.
+    @inline(__always)
+    @available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+    public static func sum<U>(_ vector: U) -> (elementsSum: Float, squaresSum: Float)
```

Comment:
I don't think you want to give this the name `sum`, because then you need to disambiguate at every use site.

Reply:
How does `sumAndSumOfSquares` sound? Should I apply the same rule to functions that return a value and index (e.g. should `maximumMagnitude(...) -> (Int, Float)` be called `maximumMagnitudeWithIndex`?

Reply:
For the latter, I would just name it `indexOfMaximumMagnitude`, but return both.